timescaledb/tutorials/financial-tick-data/financial-tick-dataset.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Analyze financial tick data - Set up the dataset
+excerpt: Set up a dataset so you can query financial tick data to analyze price changes
+keywords: [tutorials, finance, learn]
+tags: [tutorials, beginner]
+---
+
+import Install from "versionContent/_partials/_cloud-installation.mdx";
+import CreateService from "versionContent/_partials/_cloud-create-service.mdx";
+import Connect from "versionContent/_partials/_cloud-connect.mdx";
+import CreateHypertable from "versionContent/_partials/_create-hypertable-twelvedata-crypto.mdx";
+import AddData from "versionContent/_partials/_add-data-twelvedata-crypto.mdx";
+
+# Create the dataset
+
+This tutorial uses a dataset that contains second-by-second stock-trade data for
+the top 100 most-traded symbols, in a hypertable named `stocks_real_time`. It
+also includes a separate table of company symbols and company names, in a
+regular PostgreSQL table named `company`.
+
```

Comment:
Maybe add a skip link to skip past the account creation steps for readers who already have Timescale Cloud set up?

Or maybe this is better: a listing of sections so readers can see all the procedures in this page on one glance. It feels a little choppy right now, with so many little procedures one after the other and no overall sense of direction.

Reply:
I was actually wondering if we can implement the folding mechanism you have working with the hyperfunction docs? What do you think?

Reply:
Sure! Try this:

```
<Collapsible heading="$HEADING">

Content goes here

</Collapsible>
```

If you want to control the heading level, you can also add `headingLevel={3}` or whatever (defaults to 2)